cacheops/cleaner.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from .redis import redis_client
+from .sharding import get_prefix
+
+logger = logging.getLogger(__name__)
+
+
+def clear_stale_cacheops_keys(chunk_size: int, min_conj_set_size: int, dry_run: bool = False):
```

Comment:
So this one will be `reap_conjs()` or smth. Also may move default here from the command.

Reply:
I think I might have to duplicate the default: if I remove the default from the command and the user doesn't specify them, then the logic will be called with `None`, which will cause an error.